### PR TITLE
core 0.26.1

### PR DESCRIPTION
**Pull request by JoGir**
* integrate core 0.26.1
* remove duplicate IdentityImportExport
* catch crashes in AndroidIndexSyncService (missing in #771)
* debounce connectivity

### CODE REVIEW

File: qabelbox/src/main/java/de/qabel/qabelbox/index/AndroidIndexSyncService.kt [hunk 1]
```diff
@@ -58,6 +60,7 @@ class AndroidIndexSyncService() : IntentService(AndroidIndexSyncService::class.j
             }
         } catch (ex: Throwable) {
             error("Error syncing with index. Action: ${intent.action}", ex)
+            crashSubmitter.submit(ex)
```
Comment (julianseeger) [inline qabelbox/src/main/java/de/qabel/qabelbox/index/AndroidIndexSyncService.kt]: where does this come from?
and where is the difference between the CrashReporter and the CrashSubmitter (especially as you seem to need both)?

Comment (audax) [inline qabelbox/src/main/java/de/qabel/qabelbox/index/AndroidIndexSyncService.kt]: crash reporter install a crash-handler into the android runtime. The crash-submitter submits an exception as a crash. This is for cases where we catch an exception, but it is still unexpected.

**Comment by audax** [comment]
We should use the new prefix-choosing algorithm here @JoGir

**Comment by JoGir** [comment]
i prefer to do this in a "refactor account/prefix stuff" PR because there are some lines to change.

* Account is current not used in android. (account name stored in preferences)
* current prefix server https://github.com/Qabel/qabel-android/blob/master/qabelbox/src/main/java/de/qabel/qabelbox/communication/PrefixServer.java

**Pull request merged**